src/test.cc

```diff
@@ -69,7 +69,7 @@ char* mkdtemp(char* name_template) {
 string GetSystemTempDir() {
 #ifdef _WIN32
   char buf[1024];
-  if (!GetTempPath(sizeof(buf), buf))
+  if (!GetTempPathA(sizeof(buf), buf))
```

Comment:
nit: since this calls GetTempPathA() directory, consider using `buf[MAX_PATH + 1]` here (see https://docs.microsoft.com/en-us/windows/win32/api/fileapi/nf-fileapi-gettemppatha)